package.json

```diff
@@ -273,13 +279,22 @@
 				{
 					"command": "cosmosDB.update",
 					"when": "resourceFilename==cosmos-document.json"
+				},
```

Comment:
Right now there's no behavior difference between `cosmos-document.json` and `cosmos-collection.json`. Let's just combine them into one. Maybe `cosmos-result.json` or `cosmos-editor.json`?

Reply:
I tried that. Looking at the use of `&&` in the view/title clauses suggested it would work, but it didn't. :(

Reply:
I'm not talking about combining the _where_ clause, I'm talking about combining the file names. You would only have one file with one update command:
```json
"editor/title": [
  {
    "command": "cosmosDB.update",
    "when": "resourceFilename==cosmos-editor.json",
    "group": "navigation"
  },
]
```